### PR TITLE
added program used to generate filtered dataset + slight modifications to the jupyter lab file

### PR DESCRIPTION
**Pull request by Richard-M-Sullivan**
Added a python file that has the code I used to compress the data and test sets.

Also, in the jupyter notebook I added a cell that loaded the compressed training data to be used in grid search under the original cell that loads the data.

In the original data loading cell I commented out the cell that loads the predictions on the test set because it was not found.

### CODE REVIEW

**Comment by PrismaWeaver** [comment]
I'm not going to sign off on merging this branch into main as it does not add to a more cohesive whole. I understand you performed this work in an attempt to sidestep the computational requirements that your computer was unable to complete, but as I said in our correspondence: you may include details as to the work you attempted in our report as it does show why feature reduction would not help the accuracy, but I don't see what purpose it serves in the final product given the clear accuracy difference between our approaches.
